### ISSUE TITLE
Phone calls are forced to loudspeaker

### ISSUE THREAD

**Issue by David-Guillot** [FairphoneCommunity/android_device_fairphone_fp3#1]
Hi there,

I am testing @mstaz build 20200413, and it's amazingly stable, I'm impressed by how you guys are already so close to be daily-usage acceptable!

The main issue is that phone calls are forced to the loudspeaker. As you @mstaz suspected on the forum, the ear speaker does not seem to be working/detected (I tried to change audio source on a Jitsi conference, nothing else than loudspeaker was showing, instead of the usual choice between loudspeaker and ear speaker).

Let me know if I can help you tackle this one 😉

**Comment by dk1978** [comment]
Hi,
thanks for the compliments!

I just tried doing a phone call and I could not confirm any problems with the loudspeaker. The recipient voice came through the ear speaker at normal volume. Would you be kind enough to post a `logcat` output with a call in progress?

**Comment by mstaz** [comment]
Hi all,

> I just tried doing a phone call and I could not confirm any problems with the loudspeaker. The recipient voice came through the ear speaker at normal volume.

Sounds great. Do you have any other changes in your repo, e.g. in the proprietary-files.txt? Seems like you removed your repo or made in private.
I noticed that the sound_trigger.primary.msm8953.so is commented out, however it seems like it can't be build from sources so far. This also removes an error message in the logs. However it still doesn't make the ear speaker working.

**Comment by dk1978** [comment]
Hi,
yes, my version of `proprietary-files.txt` does not have `sound_trigger.primary.msm8953.so` commented out, and that is likely correct. 

I have deleted my own fork to avoid confusion. I still have all my changes locally though. First question: are we all still using the [0110 dump](https://androidfilehost.com/?fid=4349826312261714249) for the proprietary files? I don't think we should move to 0111 prematurely, and definitely not without tests and consensus.

For the moment, I will do a clean build (wipe the `out` folder) and test phone functionality again.

BR,
Daniel

**Comment by mstaz** [comment]
Hi Daniel,

I checked the differences in your `proprietary-files.txt` and after some trying I found out that vendor/etc/audio/audio_policy_configuration.xml was missing, only had the vendor/etc/audio_policy_configuration.xml :man_facepalming: 
With that ear speaker works fine. :+1: 
Need to clean up, before pushing my changes.

Yes, I switched to 0111 recently, but there shouldn't be a big change in the vendor files.

BR,
Max
